data/posts/rr-governance.md

```diff
@@ -0,0 +1,109 @@
+
+React Router has been around for over 10 years under the development and oversight of [Michael](https://x.com/mjackson) and [Ryan](https://x.com/ryanflorence). When they obtained funding for Remix in 2021 and built a team around Remix, they also indirectly built a team around React Router. That team has been working on React Router for a few years now and it's time to formalize the process we've loosely been using internally for a while now.
+
+Last week in our ["Wake Up, Remix!"](./wake-up-remix) post, we announced a new [Open Governance Model](https://github.com/remix-run/react-router/blob/main/GOVERNANCE.md) for React Router. We planned to announce these in the reverse order: first the governance model so folks knew that React Router wasn't going anywhere, followed by our plans for Remix v3.
+
+However, a leak of an old version of the Remix announcement (you can go find that yourself) forced our hand and we had to get that post out quicker than anticipated. This meant we didn't get to give the new governance model proper focus and an announcement, so we're doing that now. We think it's _just as_ exciting for the web community and the future of React Router!
+
+## Motivation
+
+React Router has gone through many major evolutions in its 10+ year lifetime, many of those dictated by the evolution of React itself (i.e., the introduction of hooks). With the recent integration of [Remix v2 into React Router v7](./react-router-v7), the API surface has increased even more. And with the arrival of React 19 and [(soon) RSC support](./rsc-preview), React has taken on aspects previously handled by React Router.
+
+While we don't want to ignore new features moving forward (we have lots of ideas we want to ship!), we do want to be cognizant of the increasing surface area and focus on where we can shed some APIs to keep React Router "lean". In addition to shedding responsibility to `React`, we also see opportunities to introduce new APIs that encapsulate the behavior of multiple existing APIs so we can simplify the project by deprecating them in a future major versions.
+
+## Design Goals
+
+To ensure we're headed in the right direction we want to keep the following design goals in mind as we consider new features moving forward, and we think a more formalized process guided by a [Steering Committee](https://github.com/remix-run/react-router/blob/main/GOVERNANCE.md#steering-committee) is the best way to do that:
+
+- **Less is More**. React Router has gained a _lot_ of functionality in the past years, but with it comes a bunch of new API surface. It's time to hone in on the core functionality and aim to reduce API surface _without sacrificing capabilities_. This may come in multiple forms, such as condensing a few existing APIs into a singular API, or deprecating current APIs in favor of a new React API.
```

Comment:
> Use plural nouns after 'bunch of'

```suggestion
- **Less is More**. React Router has gained a _lot_ of functionality in the past years, but with it comes a bunch of new API surfaces. It's time to hone in on the core functionality and aim to reduce API surface _without sacrificing capabilities_. This may come in multiple forms, such as condensing a few existing APIs into a singular API, or deprecating current APIs in favor of a new React API.
```